query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/PinotBasedRequestHandler.java

```diff
@@ -118,7 +149,7 @@ public void handleRequest(
       QueryContext queryContext,
       QueryRequest request,
       QueryResultCollector<Row> collector,
-      RequestAnalyzer requestAnalyzer) {
+      RequestAnalyzer requestAnalyzer) throws Exception {
```

Comment:
Who is throwing the exception and can the be wrapped into a runtime exception?

Reply:
Removed it now and cleaned up the code a bit.